<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThresholdToolWindow</class>
 <widget class="QDialog" name="ThresholdToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thresholding tool</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>243</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Select input data</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPatient">
       <property name="text">
        <string>Patient: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSlices">
       <property name="text">
        <string>Slices: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxPatient"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxSlices"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithm parameters</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>20</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTreshold">
       <property name="text">
        <string>Threshold[%]:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxThreshold">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxFindOptimal">
       <property name="text">
        <string>Find optimal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Correction: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxCorrection">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>159</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelAngle">
       <property name="text">
        <string>Angle[Â°]: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxAngle">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDistance">
       <property name="text">
        <string>Distance[%]: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFromTop">
       <property name="text">
        <string>From top[%]: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxFromTop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>25</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDistance">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxMinAverage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Min average</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
